docs/developer/introduction.rst

```diff
@@ -65,13 +58,18 @@ to omit the test phase of the build process:
 
     $ mvn clean install -DskipTests=true
 
+To compile for Kafka 09, Kafka 10, hbase or bigtable use the respective profile `kafka09`, `kafka10`, `hbase`, or `bigtable`.
```

Comment:
Also, the ``kafka09`` and ``kafka10`` profiles will disappear when https://github.com/locationtech/geomesa/pull/1104 is merged.